test/unit/worker.js

```diff
@@ -191,6 +344,29 @@ describe('Worker', function () {
               // this is asserting taskHandler called once, but was twice
               assert.ok(taskHandler.calledOnce, 'task was called once')
               assert.ok(doneHandler.calledOnce, 'done was called once')
+              sinon.assert.callCount(monitor.increment, 5)
+              sinon.assert.calledWith(monitor.increment.firstCall, 'ponos', {
+                token0: 'command',
+                token1: 'something.command',
+                token2: 'do.something.command',
+                queue: 'do.something.command'
+              })
```

Comment:
to the earlier point about things on lines: this sample assert block is fine - everything (including the start of the object) is on one line, so I know that that's all the arguments, and the last one's an object (which is closed in-line with the `sinon.assert`). Make scanning through code so much easier